package.json

```diff
@@ -1,6 +1,7 @@
 {
   "private": true,
   "scripts": {
+    "postinstall": "lerna bootstrap",
```

Comment:
If the build is required why not do `lerna bootstrap && yarn build`

Reply:
That was mentioned in the comments. I've yet to wrap my head/ find the time to see if we can use lerna to make this compatible with both yarn and and npm, so haven't come back to make changes to this PR yet. Feel free to lend a hand if you know lerna, it would be much appreciated!